content/ja/docs/concepts/services-networking/topology-aware-hints.md

```diff
@@ -0,0 +1,105 @@
+---
+reviewers:
```

Comment:
please remove reviewers section in translated contents

Reply:
Thanks! I remove this section.